webplugin/js/app/kommunicate.custom.theme.js

```diff
@@ -33,6 +33,7 @@ function KmCustomTheme() {
                 WIDGET_SETTINGS && WIDGET_SETTINGS.secondaryColor
                     ? WIDGET_SETTINGS.secondaryColor
                     : DEFAULT_SECONDARY_BACKGROUND_COLOR;
+            var squareIcon = kommunicate._globals.iconShape == 'square' ? '.km-chat-icon-sidebox { border-radius: 10% !important; box-shadow: 0 3px 4px 1px rgb(107 104 104 / 50%) !important; } .chat-popup-widget-container--horizontal {border-radius: 12px !important;}' : null;
```

Comment:
move css to CSS file and only css class name to the element.

Reply:
Updated the CSS changes